source/frontend/src/components/propertySelector/MapClickMonitor.tsx

```diff
@@ -7,14 +9,22 @@ import { featuresetToMapProperty } from '@/utils/mapPropertyUtils';
 import { LocationFeatureDataset } from '../common/mapFSM/useLocationFeatureLoader';
 
 interface IMapClickMonitorProps {
-  addProperty: (property: LocationFeatureDataset) => void; // TODO: This should be a featureDataset
+  addProperty: (property: LocationFeatureDataset) => void;
```

Comment:
looks like the TODO was not addressed. Does it not need to be a featureDataset anymore?

Reply:
good catch - will remove the comment

Reply:
I misunderstood your comment. According to git history the TODO was there when the property was an **IMapProperty**. Then Devin refactored that callback to correctly use the LocationFeatureSet (which is what the comments is asking for). Then I noticed that the TODO didn't apply anymore and removed it.